### PR TITLE
dehydrated carp can fit in locker again

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
backstory:
- monkey cube had a non-aabb size of 0.8x0.6 which let it usually fit in crates and lockers
- dehydrated carp had one of 0.8x0.8 which, when grid was rotated even a little, made its aabb size exceed 1
- this almost always stopped you from putting fishe in locker

so this pr makes both of them 0.6x0.6, shouldn't really affect hydrating them its such a small difference

tl;dr fishops is back baby!!!!

**Media**
left: old
right: new

https://user-images.githubusercontent.com/39013340/225388491-7610a3b8-f509-4a6e-8cf4-4c20b9433d5b.mp4

- [X] I have added a video to this PR showcasing its changes ingame

**Changelog**

:cl:
- fix: Dehydrated carp will fit in lockers now.

### CODE REVIEW

**Comment by deltanedas** [comment]
@EmoGarbage404 do you have any feedback?

**Comment by EmoGarbage404** [comment]
pr seems fine but is there a reason why a aabb size of greater than 1 causes the entity to be unable to be inserted?

**Comment by deltanedas** [comment]
because EntityStorage has max size set to 1 so if width or height of aabb are greater it wont let it
so you cant store onestar/ripley/other lockers in a locker

**Review by EmoGarbage404 — APPROVED** [review]
